tools/packaging/static-build/cloud-hypervisor/build-static-clh.sh

```diff
@@ -94,8 +120,8 @@ fi
 
 if [ "${force_build_from_source}" == "true" ]; then
 	info "Build cloud-hypervisor from source as it's been request via the force_build_from_source flag"
-	build_clh_from_source
```

Comment:
If `${force_build_from_source}" == "true` then it should always build from source (`build_clh_from_source`).

Reply:
Hmm. Theoretically.  It should build from source unless there's a cached version of that.

The `force_build_from_source` was added before the whole caching thing, which makes the name confusing. :-/

Are you okay with us renaming this as a follow up?